src/ast/ddl.rs

```diff
@@ -3262,7 +3280,9 @@ pub struct CreateTrigger {
     pub referencing: Vec<TriggerReferencing>,
     /// This specifies whether the trigger function should be fired once for
     /// every row affected by the trigger event, or just once per SQL statement.
-    pub trigger_object: TriggerObject,
+    /// This is optional in some SQL dialects, such as SQLite, and if not specified, in
+    /// those cases, the implied default is `FOR EACH ROW`.
```

Comment:
```suggestion
```
Since the optionality is clear from the type and we can avoid documenting semantic implications

Reply:
I thought it was important to specify why it is optional, as otherwise other developers which are not familiar with the peculiarities of SQLite might in the future attempt to remove the `Option`. I have added a test for this case, sure, but I believe it best to put this information immediately next to the type.